### ISSUE TITLE
provide newbie documentation for usage

### ISSUE THREAD

**Issue by mckaygerhard** [adminerevo/adminerevo#56]
the releases provide adminer.php and editor.php but the most newbie dont know how to setup.. cos dont know nothing about this is a fork.. as i suggested dont fork and made as new project

please provide directions and usage for newbies in https://docs.adminerevo.org/

thanks in advance.. and if you give me corrections i can write it for

**Comment by aoirthoir** [comment]
I agree that the editor especially is quite confusing on how to get going. Good and thorough documentation would help a lot. How can folks like us that are not sure exactly how to use Adminerevo help?

**Comment by LionelLaffineur** [comment]
Dear @mckaygerhard,
I have created a discussion about it (https://github.com/adminerevo/adminerevo/discussions/57).
If you would like to contribute, it is welcome.  You can attach files to the discussion and someone of the team will add the files at the adequate place in the project.  (https://doc.adminerevo.org is automatically populated with content coming from the repository here on Github)
Regards,
L.
